components/raftstore/src/coprocessor/dispatcher.rs

```diff
@@ -308,6 +323,10 @@ where
             400,
             BoxSplitCheckObserver::new(TableCheckObserver::default()),
         );
+        registry.register_consistency_check_observer(
```

Comment:
I don't think it's necessary. It should be registered only when it's configured.